Assets/HoloToolkit/Input/Scripts/Voice/KeywordManager.cs

```diff
@@ -84,6 +84,22 @@ void OnDestroy()
             }
         }
 
+        void OnDisable()
+        {
+            if (keywordRecognizer != null)
+            {
+                StopKeywordRecognizer();
+            }
+        }
+
+        void OnEnable()
+        {
+            if (keywordRecognizer != null && RecognizerStart == RecognizerStartBehavior.AutoStart)
```

Comment:
Oh, though, if you're mainly referring to the scenario when enabling the script doesn't also turn the recognizer on, then that's true, but I'd expect that to be the behavior. I don't think enabling/disabling specifically the script is the best way to control the recognizer, but these become useful if you enable/disable the GameObject the script is on.